File: week04-2.py
#week04-2.py　是今天的 LeetCode　挑戰題 Easy 題 二合一
#LeetCode 2529. maximumCount Count of positive Integer and Negative Integer
#找出幾個正數　有幾個負數　大的那個數量回傳
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos = 0
        neg = 0  #迴圈前面，都是0
        for i in range(len(nums)):  #迴圈裡 更新它
            if nums[i] > 0 : pos += 1
            if nums[i] < 0 : neg += 1
        #迴圈後面,把答案拿來用
        return max(pos, neg)

File: week04-2.c
//　week04-2.c　是今天的　#LeetCode　挑戰題
// #LeetCode 2529. maximumCount Count of positive Integer and Negative Integer
int maximumCount(int* nums, int numsSize) {

    int pos = 0,neg = 0; //　迴圈前面，都是0

    for(int i= 0; i<numsSize; i++) {
        if( nums[i] > 0 ) pos++; //　正數
        if( nums[i] < 0 ) neg++; //　負數
    } //　迴圈中間，更新修改

    //　迴圈後面，把它拿來用
    if(pos>neg) return pos; //　正數比較多，
    else return neg; //　不然就負數
}
